850.split-ambiguities/p.yaml

```diff
@@ -5,6 +5,7 @@
 
 - { name: pal, wwwpart: palcal, setname: palcal }
 - { name: pal, wwwpart: starlink, setname: pal-astronomical }
+- { name: pal, category: utils, setname: palcal } # cygwin
```

Comment:
Better use `ruleset:` here, as utils is too generic and repository is implied anyway.